docs/src/poly_interface.md

```diff
@@ -136,18 +136,31 @@ Return the array `[s]` where `s` is a `Symbol` representing the variable of the
 polynomial ring. This is provided for uniformity with the multivariate interface, where
 there is more than one variable and hence an array of symbols.
 
-Custom polynomial types over a given ring should define the following function
-which returns the type of a polynomial object over that ring.
-
 ```julia
 dense_poly_type(::Type{T}) where T <: RingElement
 ```
 
-Return the type of a polynomial whose coefficients have the given type.
+Return the type of a polynomial whose coefficients have the given type. In our
```

Comment:
Alternatively, we could use `@docs` above to include the docstring for `dense_poly_type`.

But not necessarily in this PR, just saying